cmd/ko/filestuff.go

```diff
@@ -74,6 +111,23 @@ func enumerateFiles(fo *FilenameOptions) chan string {
 				log.Fatalf("Error enumerating files: %v", err)
 			}
 		}
+
+		// We're done watching the files we were passed and setting up watches.
+		// Now watch for changes in the watches we were set up and stream back
```

Comment:
I can't parse this, should it be: "in the watches we set up"?

Reply:
WHO WATCHES THE WATCHERS!?!?!

I rephrased